### PR TITLE
Add zap logger for master-controller-manager

### LINKED ISSUE #3590: Refactor: replace glog for zap logger in master controller manager
Use zap logger in the master controller manager

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: replace glog for zap logger in master controller manager

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3590 

```release-note
NONE
```

### CODE REVIEW

File: api/cmd/master-controller-manager/controllers.go [hunk 1]
```diff
@@ -39,7 +39,8 @@ var allControllers = map[string]controllerCreator{
 func createAllControllers(ctrlCtx *controllerContext) (map[string]runnerFn, error) {
 	controllers := map[string]runnerFn{}
 	for name, create := range allControllers {
-		glog.V(2).Infof("Creating %s controller", name)
+		logger := log.Logger.With("controller", name)
+		logger.Info("Creating controller")
```
Comment (floreks) [inline api/cmd/master-controller-manager/controllers.go:43]: Will it also print the name of the controller?

Comment (zreigz) [reply api/cmd/master-controller-manager/controllers.go:43]: indeed, like this:
`2019-06-10T11:28:40.988+0200	info	master-controller-manager/controllers.go:43	Creating controller	{"controller": "ServiceAccounts"}`

Comment (floreks) [reply api/cmd/master-controller-manager/controllers.go:43]: Great.

**Pull request merged**